linode/manager
# Copy IP from Kubernetes Node Pool

## Description 📝

Makes each row in a node pool more consistent with the Linode instance rows including changing the color of the active row and allowing copy of the IP address for a given node.

## Preview 📷

<img width="1284" alt="Screen Shot 2022-10-25 at 5 57 31 PM" src="https://user-images.githubusercontent.com/41200331/197910442-75d1b626-9b4f-4623-be8a-0a035e3f2fcf.png">

## How to test 🧪

Confirm you can now copy the IP address of a selected node from a Kubernetes node pool
